governance.md

```diff
@@ -0,0 +1,54 @@
+
+**TSC Member** - a **Contributor** who also regularly attends and has voting rights at Technical Steering Committee meetings.
+
+## Structures
+
+We will form a **Technical Steering Committee**. The TSC shall be composed of 5 Contributors. The initial TSC will include:
+* Ali Javadi (IBM Quantum)
+* Lev S Bishop (IBM Quantum)
+* Blake Johnson (IBM Quantum)
+* Steven Heidel (AWS)
+* TBD (another non-IBM member, to be selected by the TSC)
+
+TSC meetings shall be open to all **Contributors**. The TSC will make decisions by simple majority vote of its members. TSC meetings shall require a quorum of 60% of voting members present to make a decision by vote (ie, 3 members for the initial TSC). If a TSC meeting does not have quorum, it may still discuss and record minutes. The TSC may make decisions outside of meetings (e.g. on a pull request) so long as voting/approval in that forum meets the quorum requirement. Upon request from any TSC member, any decision must be deferred until the next TSC meeting, to enable live discussion.
+
+The TSC responsibility is to balance a desire to allow a voice to the broad OpenQASM community; with lightweight processes to enable moving fast and maintaining a consistent philosophy for the language. Thus, for example, this governance document does give the freedom to the TSC to make decisions without calling a meeting and explicitly involving contributors, but the idea is that the TSC should make use of this power judiciously. The TSC has authority to make all decisions regarding OpenQASM (including amendments to this governance document), but should apply more process and seek more community input on constitutional changes than for simple fixes to the spec.
+
+The TSC may have a **Secretary** (non-voting **Contributor**) who organizes the meeting (e.g. prepares and distributes the agenda and collects a set of issues to be considered). The TSC **Secretary** will be appointed by vote of the TSC.
+* Zach Schoenfeld (IBM Quantum) is nominated to serve as the first TSC **Secretary**.
+
+The TSC will, from time to time, elect to create **Working Groups** to study issues. The TSC will appoint a **Contributor** as chair for each WG. These WGs should come back to the TSC with a proposal in the form of a language RFC or pull request to the OpenQASM specification. A WG will be automatically disbanded upon acceptance or (final) rejection of an RFC. A potential set of initial Working Groups might include:
+* Pulse grammar
+* Types and casting (including implicit casts)
+* Generics, circuit families, and subroutines
+* Defcal for non-unitaries
+
+## Open Questions
+
+* How will new **TSC Members** be selected when current **TSC Members** step down?
+    - Suggestion: nomination power to reside with TSC, voting open to all **Contributors**
+* Do we need term limits? The various source models we considered typically use a 2 year term.
```

Comment:
Maybe a term of office, and not a term limit? Eg., A TSC member is appointed for two-year term and possibly reelected/affirmed at the end of this term to continue another two-year term. However, there is no limit to the number of terms a TSC member may serve?

Reply:
Yes, a term of office would be better.

Reply:
I think this suggestion makes sense...

Reply:
If we do terms, I suggest staggering the initial terms so that the election process isn't choosing an entire TSC at once.